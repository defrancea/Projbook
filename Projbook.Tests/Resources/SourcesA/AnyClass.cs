namespace Projbook.Tests.Resources.SourcesA
{
    public class AnyClass
    {
    }
}