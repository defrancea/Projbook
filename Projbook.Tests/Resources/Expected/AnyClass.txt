// Something here

namespace Projbook.Tests.Resources.SourcesA
{
    public class AnyClass
    {
        // Something here

        // Something here
    }
}